plugins/woocommerce/tests/api-core-tests/tests/products/product-list.test.js

```diff
@@ -15,6 +15,56 @@ test.describe( 'Products API tests: List All Products', () => {
 	let sampleData;
 
 	test.beforeAll( async ( { request } ) => {
+		const createSampleData = async () => {
```

Comment:
A lot of changes in the original PR were made because of the linter. In the case of this file, I just moved this method and renamed a few variables.